homeassistant/components/sonos/speaker.py

```diff
@@ -637,6 +645,15 @@ def update_groups(self) -> None:
         """Update group topology when polling."""
         self.hass.add_job(self.create_update_groups_coro())
 
+    def update_group_for_uid(self, uid: str) -> None:
+        """Update group topology if uid is missing."""
+        if uid in self._group_members_missing:
```

Comment:
Could reverse the condition and return early, but it doesn't save you much indent

Reply:
Saves a tiny amount, but perhaps more importantly maintains this convention used in many other methods. Will adjust. 👍